shared/util.go

```diff
@@ -117,7 +117,7 @@ func ReadToJSON(r io.Reader, req interface{}) error {
 }
 
 func GenerateFingerprint(cert *x509.Certificate) string {
-	return fmt.Sprintf("% x", sha256.Sum256(cert.Raw))
+	return fmt.Sprintf("%x", sha256.Sum256(cert.Raw))
```

Comment:
The client doesn't store the server fingerprint, it stores the whole server certificate, so the change in format won't affect it.
The server has both the fingerprint and the certificate, so as long as everything on the server is consistent (which the DB update will do), all should be fine.